intro_to_r/intro_to_r.Rmd

```diff
@@ -66,7 +82,7 @@ boys to newborn girls, so he gathered the baptism records for children born in
 London for every year from 1629 to 1710.  We can take a look at the data by 
 typing its name into the console.
 
-```{r view-data, eval=FALSE}
+```{r view-data}
 arbuthnot
```

Comment:
Also not sure which of `data()` and `rm()` I like better.   But I think the complete restart (or wiping out the entire working environment with RStudio's little broom) is too much in some situations, so knowing one of `data()` or `rm()` is still useful.